packages/remark-iframes/src/index.js

```diff
@@ -127,3 +210,18 @@ function computeThumbnail (provider, url) {
   }
   return thumbnailURL
 }
+
+async function fetchEmbed (url) {
+  return fetch(url, {timeout: 1500})
+    .then((res) => res.json())
+    .then((oembedRes) => {
+      const oembedUrl = oembedRes.html.match(/src="([A-Za-z0-9_/?&%=:.]+)"/)[1]
```

Comment:
What about:
```suggestion
      const oembedUrl = oembedRes.html.match(/src="([^"]+)"/)[1]
```

Reply:
We can do that if you want, but there's no need to exclude ", we could just:

```javascript
const oembedUrl = oembedRes.html.match(/src="(.+?)"/)[1]
```

Reply:
Nice, let's do this. I cannot apply the suggestion because it's your branch ;)